src/client/Microsoft.Identity.Client/ILongRunningWebApi.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Methods for long running processes in web APIs
+    /// </summary>
+    public interface ILongRunningWebApi
+    {
+        /// <summary>
+        /// Acquires an access token for this web API from the authority configured in the application,
+        /// in order to access another downstream protected web API on behalf of a user using the OAuth 2.0 On-Behalf-Of flow.
+        /// See https://aka.ms/msal-net-on-behalf-of.
```

Comment:
Shouldn't it be https://aka.ms/msal-net-long-running-obo ?

Reply:
Yes, updated.